### PR TITLE
Graph feedback

### PR DESCRIPTION
**Pull request by lubej**
1. (Desktop) ParaTime should directly navigate when zoomed out.
2. When zoomed in the user should be able to see that status of the ParaTime.
3. When zoomed into a specific ParaTime any unrelated ParaTimes should have 50% opacity.
4. The outer stroke should be display on top of the graph content.
5. Reduce opacity on mobile tooltip.
6. Fix ‘Coming soon’ label displayed on mobile.
7. Shadows cut off on testnet graph.
8. Add selected network label.

### CODE REVIEW

**Comment by buberdds** [comment]
1) Graph position if wrong. It's moved to right and bottom and it overlaps Selected Network
2) Emerald is broken on mobile (it should behave like Cipher)

File: src/app/pages/HomePage/Graph/Graph/index.tsx [hunk 1]
```diff
@@ -218,24 +247,46 @@ const GraphCmp: ForwardRefRenderFunction<SVGSVGElement, GraphProps> = (
...
+    setOutOfDateMap({})
+  }, [network])
 
-  const disabledMap: Record<Layer, boolean> = {
-    [Layer.emerald]: isLayerDisabled(Layer.emerald),
-    [Layer.consensus]: isLayerDisabled(Layer.consensus),
-    [Layer.cipher]: isLayerDisabled(Layer.cipher),
-    [Layer.sapphire]: isLayerDisabled(Layer.sapphire),
-  }
+  const isLayerDisabled = useCallback(
```
Comment (buberdds) [inline src/app/pages/HomePage/Graph/Graph/index.tsx:266]: Why we need to wrap `getEnabledLayersForNetwork` with useCallback when disabledMap is going to be memoized?

File: src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx [hunk 2]
```diff
@@ -212,7 +219,13 @@ const ParaTimeSelectorCmp: FC<ParaTimeSelectorProps> = ({
...
   }
 
-  const isZoomedIn = scale > 1.005
+  const clearSelectedLayer = () => {
+    if (selectedLayer?.current) {
+      setSelectedLayer(undefined)
+    }
+  }
+
+  const isZoomedIn = scale > 1.07
```
Comment (buberdds) [inline src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx:228]: If it's not a problem can you add a comment what 1.07 means and why it's correct? just no idea why 1.005 is wrong and 1.07 fixes something.

File: src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx [hunk 3]
```diff
@@ -126,10 +126,17 @@ const ZoomOutBtnFade = styled(Fade)(() => ({
...
+const QuickPinchZoomOuter = styled('div')(({ theme }) => ({
   '> div': {
     position: 'absolute',
     inset: 0,
+    border: '10px solid transparent',
+    borderRadius: '50%',
+  },
+  [theme.breakpoints.up('sm')]: {
+    '> div': {
+      border: '1.7cqmin solid transparent',
```
Comment (buberdds) [inline src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx:138]: How this value is computed, where we defined a container?

File: src/app/pages/HomePage/Graph/Graph/index.tsx [hunk 4]
```diff
@@ -218,24 +247,46 @@ const GraphCmp: ForwardRefRenderFunction<SVGSVGElement, GraphProps> = (
...
+    () => ({
+      [Layer.emerald]: isLayerDisabled(Layer.emerald),
+      [Layer.consensus]: isLayerDisabled(Layer.consensus),
+      [Layer.cipher]: isLayerDisabled(Layer.cipher),
+      [Layer.sapphire]: isLayerDisabled(Layer.sapphire),
+    }),
+    [isLayerDisabled],
+  )
+
+  const enabledLayers: Layer[] = useMemo(
```
[the remark this answers is not in the document]
Comment (lubej) [reply src/app/pages/HomePage/Graph/Graph/index.tsx:283]: It is used on multiple places in the graph, and it just an abstraction, to not repeat the function call.

File: src/app/pages/HomePage/Graph/Graph/index.tsx (hunk 1 above)
Comment (lubej) [reply src/app/pages/HomePage/Graph/Graph/index.tsx:266]: Due to network dependency. Status call depends on disabledMap, which I want to trigger only when network changes.

File: src/app/pages/HomePage/Graph/Graph/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (buberdds) [reply src/app/pages/HomePage/Graph/Graph/index.tsx:283]: but why  we need `Object.keys(disabledMap).filter(layer => !disabledMap[layer as Layer]).map(layer => layer as Layer),` to get `enabledLayers` when `RouteUtils.getEnabledLayersForNetwork(network)`  returns it?

**Review by buberdds — APPROVED** [review]

**Pull request merged**